### DISCUSSION TITLE
Cutting to scale off as well as home position issues

### DISCUSSION THREAD

**Discussion by mthmi** [arkypita/LaserGRBL#1811] [Q&A]
New to this and having a few issues with m y engraver. All I know about it is that it's a generic model DVP6550 from Banggood with about a 65cmx55cm working area.

I am having issues with it going to a home position or starting from home when cutting in LaserGRBL. I thought home was the front left corner but that doesn't seem to be the case. How do I set the home position as well as start all cutting from the home position instead of it going out into the middle of the work area?

My second questions is related to scaling. I am getting files from a scanned PDF that is cleaned up, then traced in Inkscape and saved as an SVG or use the JPG. When I bring them into LaserGRBL and vectorize them there are two lines produced. One on the inside of the part line, and one on the outside so the cutter is trying to cut a part inside a part. How can I eliminate that?

**Comment by Tribo-2dot0** [comment]
For the Homing question see $3 and $23 :
https://github.com/grbl/grbl/wiki/Configuring-Grbl-v0.9

As for the double line, vectorization will always consider the line as an area and will go around both sides of it... or you fill the shape (for example: the letter "O" will have a line on the outside and another on the inside, if if you just want one fill in the "O" to make it look like a disk), either you use Center Line... or save it as a bitmap.
